SUPPORT.md

```diff
@@ -0,0 +1,7 @@
+# Support
+
+In this document you will find some generic useful informations about Dyna&omega;o:
+
+* [Dyna&omega;o website](https://dynawo.github.io/)
+* [Email contact](mailto:rte-dynawo@rte-france.com)
```

Comment:
You should also mention that you can open a ticket, and a link toward documentation IMHO

Reply:
I meant dynawo documentation, but contribution file is a good idea too :)